app/population/index.js

```diff
@@ -22,6 +26,17 @@ router.get('/', redirectBaseUrl)
 
 dailyRouter.get('/', daily)
 
+const editConfig = {
+  controller: FormWizardController,
+  name: 'edit-population',
+  templatePath: 'population/views/edit/',
+  template: '../../../form-wizard',
+  journeyName: 'edit-population',
+  journeyPageTitle: 'actions::create_population',
+}
+
+dailyRouter.use('/edit', wizard(editSteps, editFields, editConfig))
```

Comment:
I think it might be useful to use the 4th argument of this method to specify a unique value for the wizard. 

I think with the default setup it will suffer from what we experienced with the other instances. Where if you were to open multiple edit pages in different tabs they will be sharing the same session and journey models. This will lead to errors/values being shared across tabs incorrectly.